src/editor/editloop.cpp

```diff
@@ -1189,11 +1239,17 @@ static void EditorCallbackButtonDown(unsigned button)
 					if (TileToolNoFixup) {
 						Editor.ShownTileTypes.clear();
 						Editor.SelectedTileIndex = -1;
-						for (size_t i = 0; i < Map.Tileset->getTileCount(); i++) {
-							const CTileInfo &info = Map.Tileset->tiles[i].tileinfo;
-							if (Map.Tileset->tiles[i].tile) {
-								Editor.ShownTileTypes.push_back(Map.Tileset->tiles[i].tile);
+
+						tile_index index = 0;
+						for (auto &currTile : Map.Tileset->tiles) {
+							
+							/// FIXME: @tim, check pls why this line was added here with commit: 2da51da
+							const CTileInfo &info = currTile.tileinfo; 
```

Comment:
Looking at the commit, I think this is just a leftover. Can be removed.

Reply:
Fixed.